metrics/session.go

```diff
@@ -178,45 +178,54 @@ var (
 		}, []string{LblType, LblPhase})
+
+	ResourceGroupQueryTotalCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "tidb",
+			Subsystem: "session",
+			Name:      "resource_group_query_total",
+			Help:      "Counter of the total number of queries for the resource group",
+		}, []string{LblResourceGroupName})
 )
 
 // Label constants.
 const (
-	LblUnretryable    = "unretryable"
-	LblReachMax       = "reach_max"
-	LblOK             = "ok"
-	LblError          = "error"
-	LblCommit         = "commit"
-	LblAbort          = "abort"
-	LblRollback       = "rollback"
-	LblType           = "type"
-	LblDb             = "db"
-	LblResult         = "result"
-	LblSQLType        = "sql_type"
-	LblCoprType       = "copr_type"
-	LblGeneral        = "general"
-	LblInternal       = "internal"
-	LblTxnMode        = "txn_mode"
-	LblPessimistic    = "pessimistic"
-	LblOptimistic     = "optimistic"
-	LblStore          = "store"
-	LblAddress        = "address"
-	LblBatchGet       = "batch_get"
-	LblGet            = "get"
-	LblLockKeys       = "lock_keys"
-	LblInTxn          = "in_txn"
-	LblVersion        = "version"
-	LblHash           = "hash"
-	LblCTEType        = "cte_type"
-	LblAccountLock    = "account_lock"
-	LblIdle           = "idle"
-	LblRunning        = "executing_sql"
-	LblLockWaiting    = "waiting_for_lock"
-	LblCommitting     = "committing"
-	LblRollingBack    = "rolling_back"
-	LblHasLock        = "has_lock"
-	LblPhase          = "phase"
-	LblModule         = "module"
-	LblRCReadCheckTS  = "read_check"
-	LblRCWriteCheckTS = "write_check"
+	LblUnretryable       = "unretryable"
+	LblReachMax          = "reach_max"
+	LblOK                = "ok"
+	LblError             = "error"
+	LblCommit            = "commit"
+	LblAbort             = "abort"
+	LblRollback          = "rollback"
+	LblType              = "type"
+	LblDb                = "db"
+	LblResult            = "result"
+	LblSQLType           = "sql_type"
+	LblCoprType          = "copr_type"
+	LblGeneral           = "general"
+	LblInternal          = "internal"
+	LblTxnMode           = "txn_mode"
+	LblPessimistic       = "pessimistic"
+	LblOptimistic        = "optimistic"
+	LblStore             = "store"
+	LblAddress           = "address"
+	LblBatchGet          = "batch_get"
+	LblGet               = "get"
+	LblLockKeys          = "lock_keys"
+	LblInTxn             = "in_txn"
+	LblVersion           = "version"
+	LblHash              = "hash"
+	LblCTEType           = "cte_type"
+	LblAccountLock       = "account_lock"
+	LblIdle              = "idle"
+	LblRunning           = "executing_sql"
+	LblLockWaiting       = "waiting_for_lock"
+	LblCommitting        = "committing"
+	LblRollingBack       = "rolling_back"
+	LblHasLock           = "has_lock"
+	LblPhase             = "phase"
+	LblModule            = "module"
+	LblRCReadCheckTS     = "read_check"
+	LblRCWriteCheckTS    = "write_check"
+	LblResourceGroupName = "name"
```

Comment:
```suggestion
	LblName = "name"
```
Other "name" labels can share the same const